recipes/lvis/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "lvis" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lvis-{{ version }}.tar.gz
+    sha256: 55aeeb84174abea2ed0d6985a8e93aa9bdbb60c61c6db130c8269a275ef61a6e
+  - path: requirements.txt
```

Comment:
Looks like the project isn't really maintained but could you maybe open an issue at https://github.com/lvis-dataset/lvis-api/issues to mention that both `LICENSE` and `requirements.txt` are missing from the sdist?

Reply:
Done: https://github.com/lvis-dataset/lvis-api/issues/33